app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):

```

Comment:
In Python, it's more efficient not to compare a variable to `None` or `0` if there is no need. You can simply write `if not visitor.get("vaccine"):`.

Reply:
That's how it's written